README.md

```diff
@@ -1 +1,15 @@
-Hello world
+# Status Core Dev Meetings
+
+### Previous Meetings
+ №  | Date | Notes | Recording |
+ ---|---|---|---|
+ 10 | Mon, January 14, 2019 at 13:00 UTC | [notes](notes/meeting10.md) | [video](https://www.youtube.com/watch?v=4vWaqa38Hrs)
+ 9  | Mon, December 17, 2018 at 13:00 UTC | [notes](notes/meeting9.md), [agenda](notes/meeting9-agenda.md) | [video](https://www.youtube.com/watch?v=HVylXrDBJHg)
+ 8  | Mon, December 3, 2018 at 12:00 UTC | [notes](notes/meeting8.md), [agenda](notes/meeting8-agenda.md) | [video](https://www.youtube.com/watch?v=hTOuWU4Qpqo)
+ 7  | Mon, November 19, 2018 at 12:00 UTC | [notes](notes/meeting7.md) | [video](https://www.youtube.com/watch?v=mOeUN_T4So4)
+ 6  | Mon, October 15, 2018 at 12:00 UTC | [notes](notes/meeting6.md) | [video](https://www.youtube.com/watch?v=YLkk3lqf_XI)
+ 5  | Mon, October 1, 2018 at 12:00 UTC | [notes](notes/meeting5.md) | [video](https://www.youtube.com/watch?v=86vSfFnZcmA)
+ 4  | Mon, September 17, 2018 at 12:00 UTC | [notes](notes/meeting4.md) | [video](https://www.youtube.com/watch?v=p01rnaEc3B0)
+ 3  | Mon, September 3, 2018 at 12:00 UTC | [notes](notes/meeting3.md) | [video](https://www.youtube.com/watch?v=Mb58gDHWFIg)
+ 2  | Mon, August 20, 2018 at 12:00 UTC | [notes](notes/meeting2.md) | [video](https://www.youtube.com/watch?v=4r-F9SkriEo)
+ 1  | Mon, August 6, 2018 at 12:00 UTC | [notes](notes/meeting1.md) | [video](https://www.youtube.com/watch?v=P_jo4Y39tRk)
```

Comment:
In order to make sure sorting works, could we either:

- ensure notes are double digits, e.g. `notes/meeting01.md`
- alternative would be to add date in name, e.g. `notes/180806.md`

Either would work imo